transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClient.kt

```diff
@@ -37,8 +37,15 @@ interface MessagingClient {
          *
          * @property connected true if session has been configured and connection is established.
          * @property newSession indicates if configured session is new. When configuring an existing session, [newSession] will be false.
+         * @property readOnly represents the session state. When [readOnly] is set to true, the user will not be able to send or receive any new messages,
+         * but they will still be able to fetch and read existing message history.
+         * When [readOnly] is set to false, the user will be able to send and receive messages.
          */
-        data class Configured(val connected: Boolean, val newSession: Boolean) : State()
+        data class Configured(
```

Comment:
I'm supposing this API change will necessitate a change our SDK's major version number?

Reply:
Not exactly. It does not modify or break any existing behaviour. Instead, it is just extend it by providing additional information. User can decide to ignore this field, therefore it does not considered to be a breaking change.